### ISSUE TITLE
Affiliations to be keyboard navigable and screen reader friendly

### ISSUE THREAD

**Issue by discodavey** [libero/editor#503]
## Description
Affiliations are not being read out by screen readers. Keyboard tabs skip list items and move to the edit buttons.

## Tasks
- [ ] Affiliations section text to be navigable for keyboard tab and readable by screen reader.
- [ ] Next tab key jumps to affiliation list item followed by edit button.

## DoD
- [ ] Section label and affiliation list items to be keyboard navigable and readable by screen readers.

**Comment by NickDuf** [comment]
@discodavey would you mind checking my tasks? What should be readout by each list item? Does each list item require an aria tag?

**Comment by discodavey** [comment]
Looks good to me.
